src/screens/DataTableGroupBy.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Data, DataTableGroupBy, Toolbar } from 'grommet';
+import Page from '../components/Page';
+import Item from './Components/Item';
+import {
+  ComponentDoc,
+  Properties,
+  Property,
+  PropertyValue,
+  Description,
+  Example,
+} from '../components/Doc';
+
+const DataTableGroupByPage = () => (
+  <Page>
+    <ComponentDoc
+      name="DataTableGroupBy"
+      stable
+      availableAt={[
```

Comment:
Can we add a storybook link here? https://storybook.grommet.io/?path=/story/data-datatablegroupby-simple--simple

Reply:
good catch, done